src/poliastro/czml/extract_czml.py

```diff
@@ -264,10 +285,147 @@ def extract(self, ext_location=None):
+            Fill Color in rgba format
+        label_outline_color: list (int)
+            Outline Color in rgba format
+        label_font: str
+            Set label font style and size (CSS syntax)
+        label_text: str
+            Set label text
+        label_show: bool
+            Indicates whether the label is visible
+    """
+        i = self.gs_n
+
+        if id_name is not None:
+            self.parse_dict_tuples([i], [("name", id_name)], dict=self.gs_czml)
+        if id_description is not None:
+            self.parse_dict_tuples([i], [("name", id_description)], dict=self.gs_czml)
+
+        if label_fill_color is not None:
+            self.parse_dict_tuples(
+                [i, "label", "fillColor"],
+                [("rgba", label_fill_color)],
+                dict=self.gs_czml,
+            )
+        if label_outline_color is not None:
+            self.parse_dict_tuples(
+                [i, "label", "outlineColor"],
+                [("rgba", label_outline_color)],
+                dict=self.gs_czml,
+            )
+        if label_font is not None:
+            self.parse_dict_tuples(
+                [i, "label"], [("font", label_font)], dict=self.gs_czml
+            )
+        if label_text is not None:
+            self.parse_dict_tuples(
+                [i, "label"], [("text", label_text)], dict=self.gs_czml
+            )
+        if label_show is not None:
+            self.parse_dict_tuples(
+                [i, "label"], [("show", label_show)], dict=self.gs_czml
+            )
+
+    def add_ground_station(
+        self,
+        pos,
+        id_name=None,
+        id_description=None,
+        label_fill_color=None,
+        label_font=None,
+        label_outline_color=None,
+        label_text=None,
+        label_show=True,
+    ):
+        """
+        Adds a ground station
+
+        Parameters
+        ----------
+        orbit: poliastro.Orbit
+            Orbit to be added
+        pos: list [int]
+            coordinates of ground station
+            [x y z] cartesian coordinates
+            [u v] ellipsoidal coordinates (0 elevation)
+
+        Id parameters:
+        -------------
+
+        id_name: str
+            Set ground station name
+        id_description: str
+            Set ground station description
+
+        Label parameters
+        ----------
+
+        label_fill_color: list (int)
+            Fill Color in rgba format
+        label_outline_color: list (int)
+            Outline Color in rgba format
+        label_font: str
+            Set label font style and size (CSS syntax)
+        label_text: str
+            Set label text
+        label_show: bool
+            Indicates whether the label is visible
+        """
+        if len(pos) == 2:
+            u, v = pos
+            if self.cust_prop[0]:
+                a, b = self.cust_prop[0][:1]  # get semi-major and semi-minor axises
+            else:
+                a, b = 6378137.0, 6378137.0
```

Comment:
Not for this PR, but we need to carefully think how we are going to handle ellipsoids in general. For once, the value of IAU is not consistent with WGS84 https://github.com/poliastro/poliastro/blob/master/src/poliastro/constants.py#L182-L190

The IAU Working Group on Cartographic Coordinates and Rotational Elements (WGCCRE) produces a report with constants for planetary bodies and also smaller ones, see for example https://astrogeology.usgs.gov/search/map/Docs/WGCCRE/WGCCRE2015reprint. In a future PR, I think we should include those constants in `poliastro.constants` and have a way to use them from here.

Reply:
I took the IAU from [Cesium's](https://github.com/AnalyticalGraphicsInc/cesium/blob/1.58/Source/Core/Ellipsoid.js#L241) definition to avoid any graphical discrepancies. A possible solution would be to follow poliastro's constants and default to a custom ellipsoid instead of Cesium's default globe.